scripts/src/modules/insights/accordion_functionality/convert_px_property_to_number.js

```diff
@@ -0,0 +1,3 @@
+export default function convert_px_property_to_number(property) {
+    return parseInt(property.slice(0, 2))
```

Comment:
Should this instead do a string replace of `px`  with an empty string, and then parse that as an int? This would future proof it as you aren't reliant on the number always having the same amount of digits. 

Would moving everything to REM also affect this function? With string replace you could replace both `px` or `rem` with an empty string. I'm guessing JavaScript scroll always works in pixels though so accounting for REM might not be necessary.

Reply:
Thanks @AshTNA, I've changed that function to use `replace`. I don't think we need to worry about `rem` so I believe it's fine as it is but it can be easily fixed if it turns out to be an issue later on.